sorc/ncep_post.fd/UPP_PHYSICS.f

```diff
@@ -342,6 +344,7 @@ SUBROUTINE CALRH_PW(RHPW)
         k=lm-l+1
        DO J=JSTA,JEND
         DO I=1,IM
+          if(q(i,j,k)<spval) then
```

Comment:
if(q(i,j,k)<spval .and. t(i,j,k) <spval)then

Reply:
will do